### PR TITLE
[README] Update requirements section

### PR DESCRIPTION
**Pull request by jpsim**
We forgot to do this in #269, @segiddins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ documentation, post WWDC 2014.
 
 ### Requirements
 
-* [Xcode 6.3][xcode], installed in `/Applications/Xcode.app`
-* `xcode-select -p` should print Xcode 6.3's path. If it doesn't, run
-`sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`
+* A 6.x version of [Xcode][xcode] capable of building the Swift project
+  you wish to document, installed anywhere under `/Applications`.
+  Support for Xcode 7 betas is in progress (see
+  [#261](https://github.com/realm/jazzy/pull/261)).
```
Comment (segiddins) [inline README.md]: superfluous `)`

**Comment by segiddins** [comment]
:+1: other than comment

File: README.md (hunk 1 above)
Comment (jpsim) [inline README.md]: not superfluous: `(see #261)`

**Comment by segiddins** [comment]
Lol my bad

**Pull request merged**